style.css

```diff
@@ -0,0 +1,704 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap');
+
+@media (min-width: 320px) {
+  body {
+    margin: 0;
+    overflow-x: hidden;
+  }
+  .container { margin: 52px 24px 0 24px; }
+
+  header {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+  }
+  header ul { display: none; }
+
+  header a {
+    /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
```

Comment:
- [ ] [OPTIONAL] Kindly remove this line that disables linter code check. Linter is a github action that ensures that codes are standard. Getting familiar with these practices helps you to avoid bad syntax and substandard practices.